bluepyemodel/evaluation/recordings.py

```diff
@@ -187,3 +190,38 @@ def instantiate(self, sim=None, icell=None):
         self.tvector.record(sim.neuron.h._ref_t, 0.1)  # pylint: disable=W0212
 
         self.instantiated = True
+
+
+class LooseDtRecordingStimulus(ephys.recordings.CompRecording):
+    """Stimulus recording that can be checked, but that do not records at fixed dt."""
+
+    def __init__(self, name=None, location=None, variable=None):
+        super().__init__(name=name, location=None, variable=variable)
+        # to skip checking
+        self.checked = True
+
+    def instantiate(self, sim=None, icell=None, stimulus=None):
+        """Instantiate recording."""
+        self.varvector = sim.neuron.h.Vector()
+        self.varvector.record(getattr(stimulus.iclamp, f"_ref_{self.variable}"))
+
+        self.tvector = sim.neuron.h.Vector()
+        self.tvector.record(sim.neuron.h._ref_t)  # pylint: disable=W0212
+
+        self.instantiated = True
+
+    # check if in response(), varvector has to be corrected for units, or if it is already in nA
```

Comment:
this?

Reply:
Yes, it is in nA. source: https://www.neuron.yale.edu/neuron/static/py_doc/modelspec/programmatic/mechanisms/mech.html#IClamp
I removed the comment in the latest commit.